src/mscorlib/src/System/Collections/Generic/List.cs

```diff
@@ -976,8 +976,9 @@ public void Sort(IComparer<T> comparer)
             }
             Contract.EndContractBlock();
 
-            if( _size > 0) {
+            if (_size > 0) {
                 ArraySortHelper<T>.Sort(_items, 0, _size, comparison);
+                _version++;
```

Comment:
Should the version still be incremented when the size is zero? Looks like that is what the other Sorts do e.g. the one above this